docs/tools/cli/mirrormaker.rst

```diff
@@ -0,0 +1,221 @@
+'''''''''''''''''''''''''''''''''''''''''''
+
+Creates a new Aiven for Apache Kafka MirrorMaker2 replication flow. 
+
+.. Warning::
+
+  Before creating a replication flow, an :ref:`integration <avn_service_integration_create>` needs to be created between the Aiven for Apache Kafka MirrorMaker2 service and the source and target Aiven for Apache Kafka services. 
+  
+  E.g. An integration with alias ``kafka-target-alias`` between an Aiven for Apache Kafka service named ``kafka-target`` and an Aiven for Apache Kafka MirrorMaker2 named ``kafka-mm`` can be created with::
+
+    avn service integration-create \
+      -s kafka-target              \
+      -d kafka-mm                  \
+      -t kafka_mirrormaker         \
+      -c cluster_alias=kafka-target-alias
+  
+  At most **one** replication flow can be build between any two Aiven for Apache Kafka services.
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``service_name``
+    - The Aiven for Apache Kafka MirrorMaker2 service name where to create the replication flow
+  * - ``--source-cluster``
+    - The Aiven for Apache Kafka service name to be used as source for replication
+  * - ``--target-cluster``
+    - The Aiven for Apache Kafka service name to be used as target for replication
+  * - ``replication_flow_config``
+    - JSON string or path (preceded by ``@``) to a JSON configuration file for the replication flow definition
+
+**Example:** In the service ``kafka-mm`` create a new replication flow from an Aiven for Apache Kafka service with integration alias ``kafka-source-alias`` to a service named ``kafka-target-alias`` with the following settings:
+
+* include all topics with name starting with ``my-src-topic`` (topic name patterns can be defined using `Java patterns <https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html>`_)
+* exclude all topics with name ending with ``not-include``
+* ``DefaultReplicationPolicy`` as replication policy class
+* enable MirrorMaker2 heartbeats
+* enable synching of consumer groups offset every ``60`` seconds
+
+::
+
+  avn mirrormaker replication-flow create kafka-mm \
+    --source-cluster kafka-source-alias \
+    --target-cluster kafka-target-alias \
+    '
+      {
+          "emit_heartbeats_enabled": true,
+          "enabled": true,
+          "replication_policy_class": "org.apache.kafka.connect.mirror.DefaultReplicationPolicy",
+          "source_cluster": "kafka-source-alias",
+          "sync_group_offsets_enabled": true,
+          "sync_group_offsets_interval_seconds": 60,
+          "target_cluster": "kafka-target-alias",
+          "topics": [
+              "my-src-topic.*"
+          ],
+          "topics.blacklist": [
+              ".*not-include"
+          ]
+      }
+    '
+
+``avn mirrormaker replication-flow delete``
+'''''''''''''''''''''''''''''''''''''''''''
+
+Deletes an existing Aiven for Apache Kafka MirrorMaker2 replication flow. 
```

Comment:
Every time i'm reading the name "Aiven for Apache Kafka MirrorMaker2" I feel it is soooo long, can we make all those names in bold so that readers easily identify and skip them when scanning documentation?

Reply:
moving to bold names should be done across all docs, maybe this is something to discuss in #devportal?